README.md

```diff
@@ -1,121 +1,222 @@
+to install SassDoc and run it via various build tools.
+Specify `herman` as the theme
+in your SassDoc options:
+
+```bash
+sassdoc <src> --theme herman
+```
+
+
+### SassDoc Comments
+
+Currently,
+all SassDoc/Herman annotations are written as Sass comments
+starting with `///` to differentiate documentation
+from other developer comments (`//`).
+
+```scss
+// This comment will be ignored by Herman
+/// This comment will be rendered in the documentation
+```
+
+Annotation comments can be free-floating,
+or attached to a particular Sass/CSS object –
+such as a variable, mixin, function, or selector block.
+
+```scss
+/// this is a free-floating comment
+
+/// this comment is attached to the following mixin code-block
+@mixin sample-object { … }
+```
+
+
+### Herman Annotations
+
+In addition to the core SassDoc annotations,
+our `@icons` annotation allows you to
+display SVG icons from a given folder,
+and we extend the core `@example` annotation
+to display compiled Sass/Nunjucks output
+and render sample components.
+We also provide a `@preview` annotation
+for displaying color-palettes, text and spacing sizes,
+and modular ratios, and a `@font` annotation
+for displaying font-specimens.
+
+[See the full documentation for details »][docs]
+
+[oddbird]: http://oddbird.net/
+[sassdoc]: http://sassdoc.com/
+[docs]: http://oddbird.net/herman/docs/
+
+
+## Herman Configuration Options
+
+Nest all Herman-specific options under `herman` object in SassDoc options.
+
+```yaml
+theme: herman
+herman:
+  displayColors: <...>
+```
+
+All relative paths are relative to the SassDoc config file or the `cwd`.
+
+### displayColors
+
+- Type: `Array`
+- Default: `['hex', 'rgb', 'hsl']`
+
+Configures which color value formats are shown when using the
+[`@preview color-palette` annotation](http://oddbird.net/herman/docs/demo_colors.html).
+
+### extraDocs
+
+- Type: `Array`
+- Default: `[]`
+
+Add files (parsed as Markdown) to your compiled documentation. Each value in
+the list should be an `Object` with keys `path` (relative path to the local
+file) and `name` (displayed in the compiled documentation navigation), or a
+`String` path (in which case the filename will be displayed in the navigation).
+
+### customCSS
+
+- Type: `String`
+- Default: `''`
+
+Relative path to a custom CSS file,
+which will be included in the `<head>` of rendered
+[`@example` annotations](http://oddbird.net/herman/docs/demo_examples.html).
+
+### fontpath
+
+- Type: `String`
+- Default: `''`
+
+*Required if using `@font` annotation with local font files.*
+
+Relative path to a directory containing local font files.
+
+### minifiedIcons
+
+- Type: `String`
+- Default: `''`
+
+*Required if using `@icons` annotation, or if using `@example` annotations that
+require minified SVG icons.*
+
+Relative path to a file containing minified SVG icons:
+one `svg` element with many `symbol` elements,
+generated by https://github.com/Hiswe/gulp-svg-symbols or similar.
```

Comment:
Enlinken this.

Reply:
This is parsed as Markdown, so it gets auto-linkified.